kg_microbe/transform_utils/prefixmap.json

```diff
@@ -1,4 +1,5 @@
 {
     "METPO": "https://w3id.org/metpo/",
-    "biolink": "https://biolink.github.io/biolink-model/"
+    "biolink": "https://biolink.github.io/biolink-model/",
+    "kgmicrobe.strain": "https://example.org/kgmicrobe/strain/"
```

Comment:
Prefix/namespace defined in a "prefix map" as described by curies Python library here: https://curies.readthedocs.io/en/latest/struct.html#prefix-maps